ios/FluentUI/Card/CardView.swift

```diff
@@ -319,18 +325,19 @@ open class CardView: UIView {
         static let borderRadius: CGFloat = 8.0
         static let defaultTitleNumberOfLines: Int = 1
         static let twoLineTitle: Int = 2
-        static let verticalBaseHeight: CGFloat = Constants.verticalPaddingTop + Constants.iconHeight + Constants.verticalContentSpacing + Constants.verticalPaddingBottom
-        static let horizontalBaseHeight: CGFloat = 2 * Constants.horizontalPadding
-        // Layout constants
-        static let paddingTrailing: CGFloat = 16.0
-        static let paddingLeading: CGFloat = 12.0
-        // Horizontal card layout constants
-        static let horizontalPadding: CGFloat = 6.0
         static let horizontalContentSpacing: CGFloat = 12.0
-        // Vertical card layout constants
-        static let verticalPaddingBottom: CGFloat = 8.0
-        static let verticalPaddingTop: CGFloat = 10.0
-        static let verticalContentSpacing: CGFloat = 2.0
+
+        /// Small card layout constants
```

Comment:
Is the iconHeight and iconWidth going to remain the same (24) for both sizes?

Reply:
yes it will be